misc_tools.py

```diff
@@ -66,7 +66,7 @@ def run(self):
         """
         title = "Show logfile"
         location = qlogging.logfile_path()
-        message = "Logfile location: <a href='file://%s'>%s</a>" % (location, location)
+        message = "Logfile location: <a href='file:/%s'>%s</a>" % (location, location)
```

Comment:
Dit gaat denk ik fout. Die dubbele slash is echt wel nodig:
https://en.wikipedia.org/wiki/File_URI_scheme#How_many_slashes?
Theoretisch zou het volgende wèl kunnen werken:
```suggestion
        message = "Logfile location: <a href='file:%s'>%s</a>" % (location, location)
```

Levert het op windows trouwens slash-paden op? Of zijn het backslashes?

Reply:
Op die wikipediapagina staat ook nog het volgende voorbeeld:

    file:///c:/path/to/the%20file.txt

Reply:
Ah ok, in praktijk opende Windows niet de file met de dubbele slash, maar wel met een enkele. Een triple slash zoals in het voorbeeld werkt ook.